config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root to: "home#show"
 
+  resources :sessions,  only: [:new]
```

Comment:
@tmikeschu I think we need a GET to /login (sessions#new), for the navbar, but I could be misunderstanding?